src/lib/crypto/kimchi_backend/common/bigint.ml

```diff
@@ -56,6 +58,8 @@ module Make
 
   let length_in_bytes = num_limbs * bytes_per_limb
 
+  let of_decimal_string x = Bytes.of_string x |> of_decimal_string
```

Comment:
This makes a copy, which we would like to avoid by giving the correct type above.